packages/Makefile

```diff
@@ -81,6 +80,11 @@ IOS_DOC_DIR = $(DOC_DIR)/ios
 
 .PHONY: help build_core build_core.android build_core.ios build_demo build_demo.android build_demo.ios clean clean.android clean.ios docgen docgen.android docgen.ios fail_on_dev publish publish_bridge publish_bridge.android publish_bridge.ios publish_core publish_core.android publish_core.ios publish_demo publish_demo.android publish_demo.ios re re.android re.ios test test_bridge test_bridge.android test_bridge.ios test_core
 
+## Internal variables
+
+android_gomobile_cache="$(PWD)/android/.gomobile-cache"
+ios_gomobile_cache="$(PWD)/ios/.gomobile-cache"
```

Comment:
```suggestion
android_gomobile_cache="$(ANDROID_BUILD_DIR_INT_CORE)/.gomobile-cache"
ios_gomobile_cache="$(IOS_BUILD_DIR_INT_CORE)/.gomobile-cache"
```

Reply:
Can you also remove the "## Internal variables" comment, use only uppercase for both variables and put them under the `IOS_BUILD_DIR_INT_CORE` and `ANDROID_BUILD_DIR_INT_CORE` lines?

Reply:
Done.